### DISCUSSION TITLE
Bueno empezaré con un tema muy importante y es que tengo M64 plus fz pro y cuando quiero jugar multijugador online no me da el código, si alguien tiene una solución escríbelo en el chat  Saludos y buenos días

### DISCUSSION THREAD

**Discussion by SoraLetsGo44** [mupen64plus-ae/mupen64plus-ae#961] [Q&A]
#247  ## empezaré con un tema muy importante y es que tengo M64 plus fz pro y cuando quiero jugar multijugador online no me da el código, si alguien tiene una solución escríbeme en el chat  Saludos y buenos días

**Comment by fzurita** [comment]
Para poder conseguir un código:
* Tienes que estar conectado WiFi.
* Tu router tiene que ser compatible con UPnP, no todos los routers son compatibles.
* Tu servicio de Internet tiene que dejar correr servidores.

Si alguno de estos puntos no es verdad, entonces no podrás conseguir un código.

Si las dos personas están en el mismo WiFi, no es necesario conseguir un código.
